src/app/locales/zh-cn/components.yml

```diff
@@ -0,0 +1,96 @@
+channel-button:
+    title: "关注{name}"
```

Comment:
I assume the lack of a space character around any variables is intentional.

This here is the translation for the channel button tooltip, so there's nothing to worry, but could it be possible that other translations with variables that don't have any spaces around them could cause layout issues when the variable's substitution is too long? How do automated line breaks work in Chinese?

Reply:
Based on my experience using Chinese, it is not necessary to add spaces around variables in Chinese text, as Chinese characters do not require spaces to separate them. However, adding spaces between Chinese and English or numbers may improve readability. In Chinese text, line breaks are usually done simply by starting a new line at the end of a line, except when punctuation affects the line break. Although there are usually no layout issues in Chinese text because Chinese characters are all full-width and there is no special treatment of words or phrases during line breaks, I have realized that in some cases adding spaces around variables can better emphasize their content, such as adding spaces to emphasize the channel name of the channel button tooltip.

Reply:
https://github.com/streamlink/streamlink-twitch-gui/pull/958/commits/96775803da40f4bcdcc3d35d594df95d45bc1f0d should be enough. In other cases I don't think it's better to add spaces around variable names.